stable/aws-pod-identity-webhook/templates/mutatingwebhook.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: {{ include "aws-pod-identity-webhook.fullname" . }}
```

Comment:
@micahhausler I think this must be `pod-identity-webhook`? I just checked all our clusters and this already exists in each with a webhook called `iam-for-pods.amazonaws.com`.

What is supposed to happen here?

Reply:
Judging by what the name [here](https://github.com/aws/amazon-eks-pod-identity-webhook/blob/master/deploy/mutatingwebhook.yaml#L4) I think we are supposed to overwrite the `pod-identity-webhook` one in the cluster?

Reply:
OK now I realise this chart is not to be installed into an EKS cluster because EKS provides this service by default. This chart would be for none-EKS clusters.